xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -153,6 +153,28 @@ is_brick_mx_enabled(void)
     return ret ? _gf_false : enabled;
 }
 
+gf_boolean_t
+is_brick_graceful_cleanup_enabled(void)
+{
+    char *value = NULL;
+    int ret = 0;
+    gf_boolean_t enabled = _gf_false;
+    xlator_t *this = NULL;
+    glusterd_conf_t *priv = NULL;
+
+    this = THIS;
```

Comment:
`this` can be passed as an argument, but it would even be better to pass `this->private->opts`.